src/main/java/se/sundsvall/supportmanagement/integration/db/model/ErrandNumberSequenceEntity.java

```diff
@@ -0,0 +1,90 @@
+package se.sundsvall.supportmanagement.integration.db.model;
+
+
+import java.util.Objects;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+
+@Entity
+@Table(name = "errand_number_sequence")
+public class ErrandNumberSequenceEntity {
```

Comment:
No need for municipalityId in this table?

Reply:
ErrandNumbers should most likely be unique for each namespace/municipalityId combo. Would make little sense to share the sequence between municipalities to be honest. 

So yes, need to implement municipalityId in this and make sure we find the right namespace/municipalityId comb foreach call to this.